content/docs/support-and-ops/support/support-FAQ.md

```diff
@@ -34,6 +34,19 @@ A Solutions Engineer on support shall:
 - mark the support requests that are interesting/problematic so that we can talk about them on Thursday in the AE-SA meeting. We make sure we talk about past requests and learnings across all and make sure teams are aware of the priorities.
 - create a GitHub issue for the request if it is not an issue but a question or a feature request. This will allow us to track the request and make sure it is not lost
 
+## What is expected of each team's on-support engineer (`@support-$TEAM` on Slack)?
+
+Each team is required to always have a person available to help with support requests during Business Hours (09:00 - 18:00 German time). Each team has total freedom in choosing *who* is on support. Some common setups, each one with its pros and cons, are:
+- the Solutions Architect for the team is the person always on support for the team
+- the on-call engineer for the team is also the engineer that responds to the `@support-$TEAM` mentions on Slack (this can be automated via [Ailefroide](https://github.com/giantswarm/ailefroide-app/))
```

Comment:
This link is to a private repo and therefor fails the linter. You can either set it to the exception list here: https://github.com/giantswarm/handbook/blob/main/.github/workflows/markdown-link-check.json

Or consider making it public :)